postgres/pgdump.go

```diff
@@ -342,6 +359,15 @@ func processCopyStmt(conv *internal.Conv, n nodes.CopyStmt) *copyOrInsert {
 	} else {
 		logStmtError(conv, n, fmt.Errorf("relation is nil"))
 	}
+	if _, ok := conv.SrcSchema[table]; !ok {
+		// Copy statement can be applied on inherited tables. Since we are skipping
+		// inherited tables created by CREATE TABLE statement, processing insert
+		// statement can fail. For debugging purposes we log the lookup failure
+		// if we're in verbose mode, but otherwise  we just skip these statements.
+		conv.SkipStatement(prNodes([]nodes.Node{n}))
+		internal.VerbosePrintf("Processing %v statement: table %s not found", reflect.TypeOf(n), table)
+		return nil
```

Comment:
We can't return nil here. Otherwise we won't be able to parse
the data portion of the COPY-FROM statement, and we'll
likely get stuck at this point in the pg_dump file.

See comment at top of function.

I just tested this and indeed it break file parsing.

Reply:
yes Nevin, it panics if we return nil, updated return statement